### PR TITLE
fix: Change the capitalization of Component Specs

### PR DESCRIPTION
**Pull request by MuazOthman**
- Closes [UNIFY-1757](https://cypress-io.atlassian.net/browse/UNIFY-1757)

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by lmiller1990 — APPROVED** [review]

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "pageTitle": "Specs",
     "newSpecButton": "New Spec",
     "searchPlaceholder": "Search Specs",
-    "componentSpecsHeader": "Component Specs",
+    "componentSpecsHeader": "Component specs",
```
[the remark this answers is not in the document]
Comment (MuazOthman) [reply packages/frontend-shared/src/locales/en-US.json:122]: There's a separate issue to comprehensively review the casing [UNIFY-1635](https://cypress-io.atlassian.net/browse/UNIFY-1635)
This PR is meant to address the inconsistency on the specs page when used for component testing. It is a band-aid requested by @pstakoun

[the remark this answers is not in the document]
Comment (pstakoun) [reply packages/frontend-shared/src/locales/en-US.json:122]: These should likely all be consistent, but we are waiting on confirmation on this from @ryanjwilke before doing this more holistically. This one-liner is just to make the UI not brutally inconsistent between E2E and CT.

**Review by flotwig — APPROVED** [review]
Kinda surprised this isn't covered by a `cy.percySnapshot`.

**Comment by rockhold** [comment]
> Kinda surprised this isn't covered by a `cy.percySnapshot`.

I was too, but perhaps we can ensure that creating those is handled by the broader effort coming soon in [UNIFY-1635](https://cypress-io.atlassian.net/browse/UNIFY-1635).

**Review by rockhold — APPROVED** [review]

**Pull request merged**